miio/airpurifier_miot.py

```diff
@@ -249,7 +277,118 @@ def __repr__(self) -> str:
         return s
 
 
-class AirPurifierMiot(MiotDevice):
+class AirPurifierMB4Status(BasicAirPurifierMiotStatus):
+    """Container for status reports from the air purifier."""
```

Comment:
Please add the model number presenting the device to the container docstring.

Also, add an example contents for the data structure, something like this: https://github.com/rytilahti/python-miio/blob/master/miio/airconditioner_miot.py#L178

Reply:
done